packages/user-management-widget/src/lib/widget/mixins/initMixin/initComponentsMixins/initEditUserModalMixin.ts

```diff
@@ -1,18 +1,20 @@
+import {
+  ButtonDriver,
+  ModalDriver,
+  MultiSelectDriver,
+  TextFieldDriver,
+} from '@descope/sdk-component-drivers';
+import {
+  compose,
+  createSingletonMixin,
+  createTemplate,
+} from '@descope/sdk-helpers';
+import { formMixin, loggerMixin, modalMixin } from '@descope/sdk-mixins';
 import parsePhone from 'libphonenumber-js/min';
-import { compose } from '../../../../helpers/compose';
-import { createTemplate } from '../../../../helpers/dom';
-import { createSingletonMixin } from '../../../../helpers/mixins';
-import { formMixin } from '../../../../mixins/formMixin';
-import { loggerMixin } from '../../../../mixins/loggerMixin';
-import { modalMixin } from '../../../../mixins/modalMixin';
-import { ButtonDriver } from '../../../drivers/ButtonDriver';
-import { ModalDriver } from '../../../drivers/ModalDriver';
-import { MultiSelectDriver } from '../../../drivers/MultiSelectDriver';
-import { TextFieldDriver } from '../../../drivers/TextFieldDriver';
+import { User } from '../../../api/types';
```

Comment:
Perhaps same comment as parsePhone for `User` type - most likely this is going to be shared across other widgets.

Reply:
Not sure where this can be placed
I suggest that once it will be relevant for other widgets, we will extract it and maybe add a types package
WDYT?